customproperties/customProperties.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+
+<div id='testElement'></div>
+
+<script>
+CSS.registerProperty(
+{
+  name: "--my-image",
+  syntax: "<image>",
+  inherits: false,
+  initialValue: "url()"
+});
+</script>
+
+<style>
+#testElement {
+  --my-image: url("https://www.chromium.org/_/rsrc/1438879449147/config/customLogo.gif?revision=3");
+  background-image: var(--my-image);
+  width: 200px;
+  height: 200px;
+}
+</style>
+
+<script>
+// NOTE: All "ERR" notes refer to the line below
+
+
+// ---------------------------------------------------------
+// Print computed values for background-image and --my-image
+// ---------------------------------------------------------
+console.log("Initial values from style-sheet");
+var computedStyle = getComputedStyleMap(testElement);
+// console.log("typed computed --my-image: ");
+// ERR: throws TypeError("Invalid propertyName: --my-image")
+// console.log(computedStyle.get("--my-image"));
```

Comment:
Or, is there a way to write this more test-like so that errors don't stop the other parts from running?